### PR TITLE
Bugfix/file upload fixes

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: epam-assets/theme/theme_loveship.scss [hunk 1]
```diff
@@ -738,15 +738,24 @@
 
     --uui-pagination-font: var(--font-regular);
 
+    /* DropSpot */
+
+    --uui-dropspot-border: var(--night400);
+    --uui-dropstart-border: var(--uui-primary);
+    --uui-dropstart-background: var(--uui-primary-lightest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_loveship.scss:745]:
```suggestion
    --uui-dropspot-border-dropstart: var(--uui-primary);
    --uui-dropspot-background-dropstart: var(--uui-primary-lightest);
```

File: epam-assets/theme/theme_promo.scss [hunk 2]
```diff
@@ -697,15 +697,24 @@
 
     --uui-pagination-font: var(--font-regular);
 
+    /* DropSpot */
+
+    --uui-dropspot-border: var(--night400);
+    --uui-dropstart-border: var(--uui-primary);
+    --uui-dropstart-background: var(--uui-primary-lightest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_promo.scss:704]:
```suggestion
    --uui-dropspot-border-dropstart: var(--uui-primary);
    --uui-dropspot-background-dropstart: var(--uui-primary-lightest);
```

File: epam-assets/theme/theme_loveship.scss [hunk 3]
```diff
@@ -738,15 +738,24 @@
...
-    --uui-file-card-background-loaading: var(--night400);
-    --uui-file-card-error: var(--uui-negative);
-    --uui-file-card-error-block: var(--uui-negative-darkest);
-    --uui-file-card-default: var(--uui-secondary);
-    --uui-file-card-dropstart-border: var(--uui-primary);
-    --uui-file-card-dropstart-background: var(--uui-primary-lightest);
-    --uui-file-card-link-hover: var(--uui-primary-dark);
+
+    --uui-filecard-dropstart-border: var(--uui-primary);
+    --uui-filecard-dropstart-background: var(--uui-primary-lightest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_loveship.scss:753]:
```suggestion
    --uui-filecard-border-dropstart: var(--uui-primary);
    --uui-filecard-background-dropstart: var(--uui-primary-lightest);
```

File: epam-assets/theme/theme_promo.scss [hunk 4]
```diff
@@ -697,15 +697,24 @@
...
-    --uui-file-card-background-loaading: var(--gray40);
-    --uui-file-card-error: var(--uui-negative);
-    --uui-file-card-error-block: var(--uui-negative-darkest);
-    --uui-file-card-default: var(--uui-secondary);
-    --uui-file-card-dropstart-border: var(--uui-primary);
-    --uui-file-card-dropstart-background: var(--uui-primary-lightest);
-    --uui-file-card-link-hover: var(--uui-primary-dark);
+
+    --uui-filecard-dropstart-border: var(--uui-primary);
+    --uui-filecard-dropstart-background: var(--uui-primary-lightest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_promo.scss:712]:
```suggestion
    --uui-filecard-border-dropstart: var(--uui-primary);
    --uui-filecard-background-dropstart: var(--uui-primary-lightest);
```

File: epam-assets/theme/theme_loveship.scss [hunk 5]
```diff
@@ -738,15 +738,24 @@
...
+    --uui-filecard-background: var(--night400);
+    --uui-filecard-background-loaading: var(--night400);
+    --uui-filecard-error: var(--uui-negative);
+    --uui-filecard-error-block: var(--uui-negative-darkest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_loveship.scss:757]:
```suggestion
    --uui-filecard-text-error: var(--uui-negative-darkest);
```

File: epam-assets/theme/theme_promo.scss [hunk 6]
```diff
@@ -697,15 +697,24 @@
...
+    --uui-filecard-background: var(--night400);
+    --uui-filecard-background-loaading: var(--night400);
+    --uui-filecard-error: var(--uui-negative);
+    --uui-filecard-error-block: var(--uui-negative-darkest);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_promo.scss:716]:
```suggestion
    --uui-filecard-text-error: var(--uui-negative-darkest);
```

File: epam-assets/theme/theme_promo.scss [hunk 7]
```diff
@@ -697,15 +697,24 @@
...
-    --uui-file-card-default: var(--uui-secondary);
-    --uui-file-card-dropstart-border: var(--uui-primary);
-    --uui-file-card-dropstart-background: var(--uui-primary-lightest);
-    --uui-file-card-link-hover: var(--uui-primary-dark);
+
+    --uui-filecard-dropstart-border: var(--uui-primary);
+    --uui-filecard-dropstart-background: var(--uui-primary-lightest);
+    --uui-filecard-background: var(--night400);
+    --uui-filecard-background-loaading: var(--night400);
+    --uui-filecard-error: var(--uui-negative);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_promo.scss:715]:
```suggestion
    --uui-filecard-icon-error: var(--uui-negative);
```

File: epam-assets/theme/theme_loveship.scss [hunk 8]
```diff
@@ -738,15 +738,24 @@
...
-    --uui-file-card-default: var(--uui-secondary);
-    --uui-file-card-dropstart-border: var(--uui-primary);
-    --uui-file-card-dropstart-background: var(--uui-primary-lightest);
-    --uui-file-card-link-hover: var(--uui-primary-dark);
+
+    --uui-filecard-dropstart-border: var(--uui-primary);
+    --uui-filecard-dropstart-background: var(--uui-primary-lightest);
+    --uui-filecard-background: var(--night400);
+    --uui-filecard-background-loaading: var(--night400);
+    --uui-filecard-error: var(--uui-negative);
```
Comment (siarhei-epam) [inline epam-assets/theme/theme_loveship.scss:756]:
```suggestion
    --uui-filecard-icon-error: var(--uui-negative);
```

File: uui/components/fileUpload/FileCard.module.scss [hunk 9]
```diff
@@ -2,19 +2,19 @@
...
+    border: 1px solid var(--uui-filecard-background);
 
     .file-name {
         white-space: nowrap;
     }
 }
 
 .error-card-wrapper {
-    border: 1px solid var(--uui-file-card-error);
+    border: 1px solid var(--uui-filecard-error);
```
Comment (siarhei-epam) [inline uui/components/fileUpload/FileCard.module.scss:13]:
```suggestion
    border: 1px solid var(--uui-filecard-border-error);
```

File: epam-assets/theme/theme_loveship.scss (hunk 1 above)
Comment (vik753) [reply epam-assets/theme/theme_loveship.scss:745]: fixed

File: epam-assets/theme/theme_promo.scss (hunk 2 above)
Comment (vik753) [reply epam-assets/theme/theme_promo.scss:704]: fixed

File: epam-assets/theme/theme_loveship.scss (hunk 3 above)
Comment (vik753) [reply epam-assets/theme/theme_loveship.scss:753]: fixed

File: epam-assets/theme/theme_promo.scss (hunk 7 above)
Comment (vik753) [reply epam-assets/theme/theme_promo.scss:715]: fixed

File: epam-assets/theme/theme_promo.scss (hunk 6 above)
Comment (vik753) [reply epam-assets/theme/theme_promo.scss:716]: fixed

File: epam-assets/theme/theme_loveship.scss (hunk 8 above)
Comment (vik753) [reply epam-assets/theme/theme_loveship.scss:756]: fixed

**Review by siarhei-epam — APPROVED** [review]

File: epam-assets/theme/theme_loveship.scss [hunk 10]
```diff
@@ -738,15 +738,25 @@
...
-    --uui-file-card-error-block: var(--uui-negative-darkest);
-    --uui-file-card-default: var(--uui-secondary);
-    --uui-file-card-dropstart-border: var(--uui-primary);
-    --uui-file-card-dropstart-background: var(--uui-primary-lightest);
-    --uui-file-card-link-hover: var(--uui-primary-dark);
+
+    --uui-filecard-border-dropstart: var(--uui-primary);
+    --uui-filecard-background-dropstart: var(--uui-primary-lightest);
+    --uui-filecard-background: var(--night400);
+    --uui-filecard-background-loaading: var(--night400);
```
Comment (AlekseyManetov) [inline epam-assets/theme/theme_loveship.scss:755]: it seams that we have typo here - loa**a**ding - dabble 'a',

Comment (vik753) [reply epam-assets/theme/theme_loveship.scss:755]: fixed

**Pull request merged**